CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 0.16.0
+
+## Minor changes and fixes
+
+- Improve sandcastle logging, instead of list returns single string
+- Improve action output logging, user actions are logged by default as `INFO`
```

Comment:
```suggestion
- Improve sandcastle logging, instead of a list returns a single string.
- Improve action output logging, user actions are logged by default as `INFO`.
```